test/test_html2text.py

```diff
@@ -82,9 +87,10 @@ def get_baseline_name(fn):
 
 def get_baseline(fn):
     name = get_baseline_name(fn)
-    f = codecs.open(name, mode='r', encoding='utf8')
-    out = f.read()
-    f.close()
+    with codecs.open(name, mode='r', encoding='utf8') as f:
+        out = f.read()
+    #f.close()
```

Comment:
Good eye on using `with` for files.
This commented line would be better to be removed for cleaner source code